index.html

```diff
@@ -36,7 +45,11 @@ <h1 class="h1">Hey there. I’m <br> Freddy <br> <span class="sub_text">I’m a
         <li id="angellist"><a href="#" target="_blank"><img src="./img/angellist.svg" alt="angellist icon"></a></li>
       </ul>
     </nav>
+    <button id="vector"><img src="/img/imgdes/vector_button.png" alt="vectorb"></button>
   </section>
+    <div class="work_title">
+      <h2>My Recent Works</h2>
```

Comment:
- [ ] Good job implementing the changes here. now the title of recent works is appearing but the problem is that the  font is not `Inter`. as a project requirement, we need to be stick to the figma design. so kindly consider adding `Inter` font here.
your design
![as](https://user-images.githubusercontent.com/96683843/213874317-25c0b1e3-71c2-472d-99c3-9fa06232e2bb.jpg)
figma design
![asdf](https://user-images.githubusercontent.com/96683843/213874329-bff1bec1-f494-4228-aa32-f6b760712f6f.jpg)